content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -75,6 +75,34 @@ appending a container name to the command, with a `-c` flag, like so:
 kubectl logs counter -c count
 ```
 
+Since v1.32, you can request a specific log stream of a container in a running Pod.
+This functionality requires the
+`PodLogsQuerySplitStreams` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
+Unfortunately, only the server side changes have been merged so far; the [corresponding kubectl changes](https://github.com/kubernetes/kubernetes/pull/128159) are still under review.
+In the meantime, you can still use this feature by directly accessing the API.
```

Comment:
We prefer to write documentation that is more [timeless](https://developers.google.com/style/timeless-documentation). Try this:
```suggestion
With that feature gate enabled, Kubernetes {{< skew currentVersion >}} allows access to these
log streams directly via the Pod API. You can fetch a specific stream by specifying the stream name (either `Stdout` or `Stderr`),
using the `stream` query string. You must have access to read the `log` subresource of that Pod.
```